<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>contactInfoDialog</class>
 <widget class="QDialog" name="contactInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgba(248, 253, 255, 206);
color:rgb(1, 31, 54);\n</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="topMargin">
    <number>20</number>
   </property>
   <item row="7" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <property name="topMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Email address:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="emailAddr_lineEdit">
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>255</number>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="topMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Headline</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="headline_lineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="10" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <property name="topMargin">
      <number>10</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="close_pushButton">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
color:white;
border:1px solid white;
border-radius:5px;
padding:5px;
background:rgb(68, 170, 248)
}
QPushButton:hover
{
	border:1px solid rgb(32, 74, 135);
	border-radius:5px;
	background:rgb(48, 162, 249);
}</string>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_9">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">margin-top:10px</string>
     </property>
     <property name="text">
      <string>Contact info:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="topMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Firstname</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="firstname_lineEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">background:rgba(248, 253, 255, 206);
color:rgb(1, 31, 54);\n</string>
       </property>
       <property name="text">
        <string>Lastname</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lastname_lineEdit">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="topMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Address: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="addr_plainTextEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <property name="topMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Birthday:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="bd_year_comboBox">
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>Year</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="bd_month_comboBox">
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>January</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>February</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>March</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>April</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>May</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>June</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>July</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>August</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>September</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>October</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>November</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>December</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="bd_day_comboBox">
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>11</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>12</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>13</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>14</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>15</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>16</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>17</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>18</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>19</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>20</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>21</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>22</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>23</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>24</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>25</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>26</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>27</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>28</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>29</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>30</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>31</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="topMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Country:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="country_comboBox">
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>Iran</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>United States of America</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Iraq</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Russia</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Turkey</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Germany</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>France</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="8" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <property name="topMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Link: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="link_lineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="9" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
